portals/application/configs/klear/FaxesList.yaml

```diff
@@ -137,6 +138,15 @@ production:
       class: ui-silk-eye
       disableSave: true
 
+    faxesOutView_screen:
+      <<: *faxesInOutEdit_screenLink
+      parentOptionCustomizer:
+        - IvozProvider_Klear_Options_OptionsCustomizerFilterForFax
+      title: _("View Incoming %s %s", ngettext('Faxfile', 'Faxefiles', 0), "[format| (%item%)]")
```

Comment:
Um, this is wrong actually. It should be "outgoing". I'll add the translation as you said